src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml

```diff
@@ -16,165 +16,275 @@
+			<StaticResource x:Key="FilledTextBoxCharacterSpacing"
+							ResourceKey="BodyLargeCharacterSpacing" />
+			<StaticResource x:Key="FilledTextBoxBorderThicknessNormal"
+							ResourceKey="TextBoxOutlinedStrokeHeight" />
+			<StaticResource x:Key="FilledTextBoxBorderThicknessFocused"
+							ResourceKey="TextBoxFocusStrokeWidth" />
+			<!--#endregion-->
+
+			<!--#region Delete Button Style-->
+			<x:Double x:Key="TextBoxClearGlyphWidth">20</x:Double>
+			<x:Double x:Key="TextBoxClearGlyphHeight">20</x:Double>
+			<!--#endregion-->
+
+			<!--#region Filled Text Box Style-->
+			<CornerRadius x:Key="FilledTextBoxCornerRadius">4,4,0,0</CornerRadius>
+			<macos:Thickness x:Key="FilledTextBoxPadding">16,7</macos:Thickness>
+			<not_macos:Thickness x:Key="FilledTextBoxPadding">16</not_macos:Thickness>
+			<x:Double x:Key="FilledTextBoxMinHeight">58</x:Double>
+			<x:Double x:Key="FilledTextBoxBorderHeightFocused">2</x:Double>
+			<!--#endregion-->
+
+			<!--#region Outlined Text Box Style-->
+			<x:Double x:Key="OutlinedTextBoxBorderThickness">1</x:Double>
+			<CornerRadius x:Key="OutlinedTextBoxCornerRadius">4</CornerRadius>
+			<macos:Thickness x:Key="OutlinedTextBoxPadding">16,7,14,7</macos:Thickness>
+			<not_macos:Thickness x:Key="OutlinedTextBoxPadding">16,16,14,16</not_macos:Thickness>
+			<x:Double x:Key="OutlinedTextBoxMinHeight">56</x:Double>
+			<Thickness x:Key="OutlinedTextBoxBorderPadding">1</Thickness>
+			<x:Double x:Key="OutlinedTextBoxBorderThicknessFocused">2</x:Double>
 			<!--#endregion-->
 		</ResourceDictionary>
 	</ResourceDictionary.ThemeDictionaries>
 
-	<!--#region Delete Button Style-->
-	<x:Double x:Key="MaterialTextBoxClearGlyphWidth">20</x:Double>
-	<x:Double x:Key="MaterialTextBoxClearGlyphHeight">20</x:Double>
-	<!--#endregion-->
-
-	<!--#region Filled Text Box Style-->
-	<CornerRadius x:Key="MaterialFilledTextBoxCornerRadius">4,4,0,0</CornerRadius>
-	<macos:Thickness x:Key="MaterialFilledTextBoxPadding">16,7</macos:Thickness>
-	<not_macos:Thickness x:Key="MaterialFilledTextBoxPadding">16</not_macos:Thickness>
-	<x:Double x:Key="MaterialFilledTextBoxMinHeight">58</x:Double>
-	<x:Double x:Key="MaterialFilledTextBoxBorderHeightFocused">2</x:Double>
-	<!--#endregion-->
-
-	<!--#region Outlined Text Box Style-->
-	<x:Double x:Key="MaterialOutlinedTextBoxBorderThickness">1</x:Double>
-	<CornerRadius x:Key="MaterialOutlinedTextBoxCornerRadius">4</CornerRadius>
-	<macos:Thickness x:Key="MaterialOutlinedTextBoxPadding">16,7,14,7</macos:Thickness>
-	<not_macos:Thickness x:Key="MaterialOutlinedTextBoxPadding">16,16,14,16</not_macos:Thickness>
-	<x:Double x:Key="MaterialOutlinedTextBoxMinHeight">56</x:Double>
-	<Thickness x:Key="MaterialOutlinedTextBoxBorderPadding">1</Thickness>
-	<x:Double x:Key="MaterialOutlinedTextBoxBorderThicknessFocused">2</x:Double>
-	<!--#endregion-->
-
-	<SolidColorBrush x:Key="MaterialDisabledOutlinedTextBoxBorderBrush"
-					 Opacity="0.12"
-					 Color="{ThemeResource OnSurfaceColor}" />
-
```

Comment:
For these ones, we should be keeping the `Material` prefix on the resource keys since they are specific to the Material styles, these are not keys that are shared by the Fluent styles so we should keep them named as they were